### ISSUE TITLE
Order of the companies

### ISSUE THREAD

**Issue by madoar** [freystef/Rails#4]
Hi,
In my 1835 games we discovered, that it's often quite easily forgettable in which order the train companies operate in the operation round.
I know that the table at the bottom of the map shows the order of the companies correctly but it's not always fully visible and as far as my knowledge goes I think it's also not manually displayable.
The only company table that is always shown is the one in the stock window but there the companies are ordered by their opening order.
I think it would be quite useful to add an option to sort the companies in the stock window by their current stock price, which also displays the order in which they operate.
In addition it would be nice to add a button to fully display the train company table in the map window whenever it's needed.

Greetings
Marc

**Comment by jdgalt** [comment]
> I think it would be quite useful to add an option to sort the companies in the stock window by their current stock price, which also displays the order in which they operate.

Yes!  I've always felt the company list ought to be this way.  For all games, not just 1835.

Having a second company list in the map window is redundant.  It ought not exist at all (or at least, I want the option to turn it off).
